src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_net_command_factory_json.py

```diff
@@ -398,6 +401,19 @@ def make_send_breakpoint_exception_message(self, *args, **kwargs):
     def make_process_created_message(self, *args, **kwargs):
         return NULL_NET_COMMAND  # Not a part of the debug adapter protocol
 
+    @overrides(NetCommandFactory.make_process_about_to_be_replaced_message)
+    def make_process_about_to_be_replaced_message(self):
+        event = ExitedEvent(ExitedEventBody(-1, reason="processReplaced"))
```

Comment:
Since we're adding a custom property to a standard message, let's do some namespacing here to prevent future conflicts - something like "pydevd.reason".